style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  display: flex;
+  flex-direction: column;
+  height: 480px;
+  width: 320px;
+  margin: 0;
+}
```

Comment:
[OPTIONAL] I've noticed that you used a fixed width for your whole project. Please take into account that, although this is the width displayed on the design, you shouldn't use a fixed width for your projects (it is just the width of the device used for the screenshot). I will explain: 

The idea of Responsive Web Design is that your webpage should adapt itself to any device/window size that opens it. In other words, it is responsive. This means that using a fixed width for your sections will break this responsiveness, causing a static view to be formed. So, what can you do about it? Simple, use percentages! Instead of `width: 327px`, use `width: 100%`. This will make your project stretches to any window size that you can imagine 😉. If you need a smaller section, use a lower percentage. If you really need to use a fixed width, keep in mind that it may break the responsiveness of your webpage 😄! 

_But, my mobile version looks horrible stretched to a big window 😢._ 

Don't worry about this! There is a thing called [media query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) that will allow you to add different styling to different windows sizes in the future. Then, you can edit a mobile version and a desktop version for your apps, depending on the window size 😄. But for now, you should focus on the mobile version 😉. 

With a fixed width, even on mobile screens, it doesn't fit correctly. Check your project on an iPhone SE: 
![image](https://user-images.githubusercontent.com/42799579/198755460-6abb0c94-e6d5-47f2-93ff-8767b5f94414.png)

If you are interested to learn more about it, check [this link](https://www.w3schools.com/html/html_responsive.asp) and this [youtube video](https://www.youtube.com/watch?v=VQraviuwbzU). 

Good luck and keep coding 😃!